packages/react-data-grid/style/rdg-interaction-masks.less

```diff
@@ -70,32 +71,31 @@
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
 }
 
-input.editor-main:focus, select.editor-main:focus {
+input.editor-main:focus,
+select.editor-main:focus {
   border-color: #66afe9;
   border: 2px solid #66afe9;
   background: #eee;
   border-radius: 4px;
 }
 
-.rdg-editor-container input.editor-main::-moz-placeholder, select.editor-main::-moz-placeholder {
-  color: #999999;
-  opacity: 1;
-}
-
-.rdg-editor-container input.editor-main:-ms-input-placeholder, select.editor-main:-ms-input-placeholder {
+.rdg-editor-container input.editor-main:-ms-input-placeholder,
+select.editor-main:-ms-input-placeholder {
   color: #999999;
 }
 
-.rdg-editor-container input.editor-main::-webkit-input-placeholder, select.editor-main::-webkit-input-placeholder {
+.rdg-editor-container input.editor-main::placeholder,
+select.editor-main::placeholder {
   color: #999999;
 }
 
-.rdg-editor-container input.editor-main[disabled], select.editor-main[disabled], .rdg-editor-container input.editor-main[readonly], select.editor-main[readonly], fieldset[disabled] .rdg-editor-container input.editor-main, fieldset[disabled] select.editor-main {
```

Comment:
>`fieldset[disabled] .rdg-editor-container input.editor-main`

Don't think that's possible since we use portals, right? so I removed that selector.

Reply:
I am not sure why do we even have a `fieldset` selector and with portals it is not even possible. This selector was possibly copied from `bootstrap.css`

Reply:
Also, I am not sure if we event disable the editors